cimetrics/github_pr.py

```diff
@@ -20,6 +20,10 @@
 class GithubPRPublisher(object):
     def __init__(self):
         self.env = get_env()
+        if self.env is None:
+            print("GithubPRPublisher requires valid environment")
```

Comment:
Not sure if we need the additional log line?

Reply:
That seems redundant with line 110?

Reply:
I've removed the log line. I think it's still OK to check whether self.env is valid though.